contrib/automation_tests/orbit_symbol_cache.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.symbols_tab import LoadAllSymbolsAndVerifyCache, ClearSymbolCache, LoadSymbols, ReplaceFileInSymbolCache
+from test_cases.main_window import EndSession
+
+"""Test symbol loading with and without local caching.
+
+Before this script is run there needs to be a gamelet reserved and
+"hello_ggp_standalone" has to be started. 
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test it needs
+to be run from 64 bit python.
+
+This automation script covers a basic workflow:
+ - connect to a gamelet
+ - delete the symbol cache
+ - load all symbols, store loading times, verify symbol files exist in cache
+ - restart session
+ - load all symbols again, loading times should have increased significantly
```

Comment:
Decreased or increased? 

My understanding is that you delete the cache at the start, the load symbols (takes long), restart, load symbols again (should be much faster).